src/audio/drc/Testbench_drc_lin2dB/drc_lin2dB_fixed.c

```diff
@@ -0,0 +1,137 @@
+// SPDX-License-Identifier: BSD-3-Clause
+//
+// Copyright(c) 2020 Intel Corporation. All rights reserved.
+//
+// Author: Shriram Shastry <malladi.sastry@linux.intel.com>
+
+//	drc_lin2dB_fixed.c :
+//	warning disabled to support file system
+//
+#pragma warning (disable : 4996)
+#pragma warning (disable : 4013)
+#include <stdio.h>
+#include "stdint.h"
+#include "math.h"
+#include "typdef.h"
+#include "norm.h"
+#include "testvector.h"
+#include <stdlib.h>
+#include <stddef.h>
+#include <string.h>
+int32_t drc_lin2db_fixed(int32_t linear, uint16_t idx, FILE* fd);		/* Input:Q6.26 Output:Q11.21 */
+static inline int32_t log10_fixed(int32_t x);
+
+
+
+int main()
```

Comment:
if this is a unix tool it should go under tools/. src/ should be reserved for DSP src files.

Reply:
It's a testbench containing DSP src source file . Expectation is we should re use src file function.
For any change in src function, we should be able to real function and avoid back porting ?
It will be nice if we have a better approach instead of moving under tool/.src/ .

Reply:
Yes, the testbench implements a the firmware as library and links it in to avoid code duplocation. I am trying to make it easier because the current method is rather complex and also builds the firmware in an annoying method that puts artificial restrictions on the code layout. I am pushing to prioritize #3617 as that will make reusing src/ code in tools/ much easier. 

See the following links for reference on my complaints
https://github.com/thesofproject/sof/blob/master/tools/testbench/CMakeLists.txt#L41
https://github.com/thesofproject/sof/blob/master/tools/testbench/topology.c#L83
https://github.com/thesofproject/sof/blob/master/tools/testbench/testbench.c#L42